tasks/update-cr-status/update-cr-status.yaml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: update-cr-status
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    A tekton task that updates the passed CR status with the
+    contents stored in the resultsFile.
+  params:
+    - name: resourceType
+      description: The type of resource that is being patched
+      type: string
+      default: release
+    - name: resource
+      description: The namespaced name of the resource to be patched
+      type: string
+    - name: resultsFilePath
+      description: |
+        Path to the JSON string of the results in the data workspace which will be added to the resource's status
+      type: string
+  workspaces:
+    - name: data
+      description: Workspace where the results file is stored
+  steps:
+    - name: update-cr-status
+      image: quay.io/redhat-appstudio/release-service-utils:bc81bfed6062a386e48a76b252c6f33b52c411b0
+      script: |
+        #!/usr/bin/env bash
+        set -ex
+
+        RESULTS_FILE="$(workspaces.data.path)/$(params.resultsFilePath)"
+        if [ ! -f "${RESULTS_FILE}" ] ; then
+            echo "No results JSON file was found."
+            exit 0
+        fi
+
+        if ! jq -e . >/dev/null 2>&1 "${RESULTS_FILE}" ; then
```

Comment:
Nit: I don't think the `-e` is needed here. If it's an invalid json, `jq` will return a non-zero code anyway, even without it. With `-e`, it would also return non-zero code for `null` or `false` value (which is an unlikely scenario here).

Reply:
Updated